tests/autoyast/autoyast_reboot.pm

```diff
@@ -25,8 +25,12 @@ sub run {
     reset_consoles;
 
     #obsoletes installation/autoyast_reboot.pm
-    assert_screen("bios-boot",     900);
-    assert_screen("autoyast-boot", 20);
+    assert_screen("bios-boot", 900);
+
+    if (check_var("BOOTFROM", "d")) {
+        assert_screen("inst-bootmenu", 60);
+    }
+    else { assert_screen("autoyast-boot", 20); }
```

Comment:
Actually this change is very related because autoupgrade jobs are also migration and I think they also deserves to be patched :) If autoupgrade are not migrations then I will revert changes to this file.